include/zephyr/xen/public/domctl.h

```diff
@@ -15,6 +15,10 @@
 #error "domctl operations are intended for use by node control tools only"
 #endif
 
+#ifdef __DOXYGEN__
+#define XEN_GUEST_HANDLE_64(x) x
+#endif
+
```

Comment:
please update `PREDEFINED` in doxyfile instead

Reply:
got it, done